Kconfig

```diff
@@ -216,3 +216,12 @@ menu "Test suite"
 depends on FEATURE_RUNNABLE_PROGRAMS
 source "src/test/Kconfig"
 endmenu
+
+menu "Bindings"
+config NODE_BINDINGS
+	bool "Node.js bindings"
+	depends on HAVE_NODE
+	default n
```

Comment:
if you're testing it and it works, we better `default y`.

And we should also include this in our semaphore so it builds your bindings as well.